activiti-cloud-service-common/pom.xml

```diff
@@ -30,16 +30,17 @@
     <module>activiti-cloud-services-swagger</module>
     <module>activiti-cloud-service-error-handlers</module>
     <module>activiti-cloud-services-test-security-keycloak</module>
+    <module>activiti-cloud-services-test-containers</module>
   </modules>
   <properties>
-    <activiti.version>7.1.230</activiti.version>
+    <activiti.version>7.1.235</activiti.version>
     <!-- dependencies -->
     <commons-beanutils.version>1.9.4</commons-beanutils.version>
     <commons-configuration.version>1.8</commons-configuration.version>
     <commons-lang.version>2.6</commons-lang.version>
     <guava.version>23.6-jre</guava.version>
     <java-semver.version>0.9.0</java-semver.version>
-    <keycloak.version>4.8.3.Final</keycloak.version>
+    <keycloak.version>8.0.1</keycloak.version>
     <logstash.version>4.11</logstash.version>
     <resteasy.version>3.6.1.Final</resteasy.version>
```

Comment:
resteasy should be updated to the matching version: 3.9.1.Final
as in https://search.maven.org/artifact/org.keycloak/keycloak-parent/8.0.1/pom

Reply:
done